s/shiny-server/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "maintainer":"gauravpbankar13",
+    "package_name" : "shiny-server",
+    "github_url": "https://github.com/rstudio/shiny-server",
+    "version": "v1.5.23.1030",
+    "default_branch": "master",
+    "package_dir": "s/shiny-server",
+    "build_script": "shiny-server_ubi_9.3.sh",
+    "validate_build_script": true,
+    "use_non_root_user": false,
+    "docker_build": false,
+    "v.*.*.*.*": {
```

Comment:
Should be v1.*.*.*.* or v*.*.*.*.*

Reply:
Hi @prachikurade1 :
Updated code as per requirement.